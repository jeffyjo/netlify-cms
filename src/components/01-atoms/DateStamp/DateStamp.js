import React from 'react'
import PropTypes from 'prop-types'

import './atoms.date-stamp.scss'

let DateStamp = ({ date }) => {
  return <div className="a-date-stamp">{date}</div>
}

DateStamp.propTypes = {
  date: PropTypes.string,
}

export default DateStamp
