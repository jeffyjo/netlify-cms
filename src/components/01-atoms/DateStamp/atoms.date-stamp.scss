.a-date-stamp {
  font-size: 14px;
  font-style: italic;
}