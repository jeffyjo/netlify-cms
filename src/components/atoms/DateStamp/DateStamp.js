import React from 'react'

import './atoms.date-stamp.scss'

let DateStamp = ({date}) => {
	return (
		<div className="a-date-stamp">{date}</div>
	)
}

export default DateStamp