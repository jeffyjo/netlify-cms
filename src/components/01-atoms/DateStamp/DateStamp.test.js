import React from 'react'
import renderer from 'react-test-renderer'

import DateStamp from './DateStamp'

describe('Atom | DateStamp', () => {
  test('DateStamp should render', () => {
    let component = renderer.create(
      <DateStamp date="18/02/2019" />
    )
    
    let tree = component.toJSON()

    expect(tree).toMatchSnapshot()
  })
})