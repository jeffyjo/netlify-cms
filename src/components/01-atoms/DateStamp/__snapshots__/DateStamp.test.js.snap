// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atom | DateStamp DateStamp should render 1`] = `
<div
  className="a-date-stamp"
>
  18/02/2019
</div>
`;
